src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -112,7 +112,7 @@ public static IDicomServerBuilder AddHostedServices(this IDicomServerBuilder ser
             {
                 c.ApiVersionReader = new UrlSegmentApiVersionReader();
                 c.AssumeDefaultVersionWhenUnspecified = true;
-                c.DefaultApiVersion = new ApiVersion(1, 0, "prerelease");
+                c.DefaultApiVersion = new ApiVersion(1, 0);
```

Comment:
major only

Reply:
We don't have an option to just initialize using major version. I achievable by creating a wrapper, but I don't believe that's necessary. Please feel free to comment if you feel other way.